src/section-solving-one-step-equations-involving-percentages.mbx

```diff
@@ -63,13 +64,13 @@
             <solution>
                 <p>Each question can be translated from English into a math equation by reading it slowly and looking for the right signals.<ol label="a">
                     <li>
-                        <p>The word <q>is</q> means about the same thing as the <m>=</m> sign. <q>How much</q> is a question phrase, and we can let <m>x</m> be the unknown amount (in dollars).The word <q>of</q> translates to multiplication, as discussed earlier. So we have:<me>\underbrace{\strut x}_{\text{How much}}\underbrace{\strut =}_{\text{is}}\underbrace{\strut 0.30}_{\text{30\%}}\underbrace{\strut\cdot}_{\text{of}}\underbrace{\strut 24}_{\text{\$24}}</me></p>
+                        <p>The word <q>is</q> means about the same thing as the <m>=</m> sign. <q>How much</q> is a question phrase, and we can let <m>x</m> be the unknown amount (in dollars).The word <q>of</q> translates to multiplication, as discussed earlier. So we have:<me>\underbrace{\strut x}_{\text{How much}}\underbrace{\strut =}_{\text{is}}\underbrace{\strut 0.30}_{\text{30%}}\underbrace{\strut\cdot}_{\text{of}}\underbrace{\strut 24}_{\text{\$24}}</me></p>
```

Comment:
Here, you changed '\%' to '%' because you saw that backslash appear in the HTML. But by changing it this way, the PDF will not compile at all, because the unescaped percent sign will comment out everything that follows on this line (including the closing `\]` for the math content.)

MathJax made the decision not to act on macros inside `\text{}`, because MathJax is for math content, not text content. That is why the `\%` is not executed and you see the backslash.

Given all this information, I think that the best thing to do is change the `\text{30\%}` to `\text{30}\%`. (If you don't put the "30" in `\text{}`, its styling clashes with the other text.) This may make the percent sign look weird compared to the rest of the text. But I can't think of a better option.

I will do this right here on GitHub, but you should be aware of this for future situations.